vendor/github.com/openshift/library-go/pkg/manifestclient/write_roundtripper.go

```diff
@@ -215,6 +215,9 @@ func (mrt *writeTrackingRoundTripper) roundTrip(req *http.Request) ([]byte, erro
 	ret.SetNamespace(serializedRequest.ActionMetadata.Namespace)
 	if actionHasRuntimeObjectBody { // TODO might be able to do something generally based on discovery if absolutely necessary
 		ret.SetGroupVersionKind(bodyObj.GetObjectKind().GroupVersionKind())
+	} else 
+	{
+		ret.SetGroupVersionKind(schema.GroupVersionKind{Group: gvr.Group, Version: gvr.Version, Kind: "Authentication"})
```

Comment:
kube returns the patched/updated object (full obj) in the response, so we need at least an empty GVK. We could use discovery for this, wdyt?